github-merge-flow.jsonc

```diff
@@ -0,0 +1,10 @@
+// IMPORTANT: This file is read by the merge flow from main branch only. 
+{
+    "merge-flow-configurations": {
+        // Automate opening PRs to merge aspire release/8.0-preview5 branch back to release/8.0
+        "release/8.0-preview5":{
+            "MergeToBranch": "release/8.0",
+            "ExtraSwitches": "-QuietComments"
+        }
```

Comment:
preview5 is no longer active.

cc @joperezr

Reply:
Thank you for letting us know! 
That was the only usage of: github-dnceng-branch-merge-pr-generator action in 
https://github.com/dotnet/versions/blob/3144a00684544208a7037c79ddbbecca78c4cc59/Maestro/subscriptions.json#L625-L640
by dotnet/aspire.